src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1146,17 +1154,14 @@ protected function processChargebacks( WC_Order $order, $payment ) {
 			//
 
 			// New order status
-			$new_order_status = self::STATUS_ON_HOLD;
-
-			// Overwrite plugin-wide
-			$new_order_status = apply_filters( Mollie_WC_Plugin::PLUGIN_ID . '_order_status_on_hold', $new_order_status );
-
-			// Overwrite gateway-wide
-			$new_order_status = apply_filters( Mollie_WC_Plugin::PLUGIN_ID . '_order_status_on_hold_' . $this->id, $new_order_status );
-
-			$payment_method_title = $this->getPaymentMethodTitle( $payment );
+            $new_order_status = self::STATUS_ON_HOLD;
+            $new_order_status = $this->overwriteFilters(
```

Comment:
Why it's called `overwriteFilters`? Does not the method just apply filters? 
And why they need to stay in a different method but not the others in other methods?

Reply:
yeah, I can find a better name. I extracted it because I saw the same pattern in both processRefunds and processChargebacks methods this filters deal with the new status so I thougth as it's related to the update status they belonged with the other new methods. There are lots of other filters in the class but these were repeating and for the same reason. Do you think I should put them back in their main method?

Reply:
Not sure about the new name, it does not explain the action taken. Anyway I leaved another comment about not having the method at all.